Umbraco-Cloud/Databases/Backups/index.md

```diff
@@ -0,0 +1,62 @@
+## Backups
```

Comment:
Remember YAML tags on new articles:
```
---
versionFrom: 7.0.0
---
```

.. or versionFrom: 8.0.0 if what's in the article is **only** relevant for Umbraco 8.